src/main/scala/com.snowplowanalytics/iglu.schemaddl/migrations/Migration.scala

```diff
@@ -98,14 +98,17 @@ object Migration {
     } yield cell
   }
 
-  def buildMigrationMatrix(schemas: NonEmptyList[IgluSchema]): NonEmptyList[OrderedSchemas] =
+  def buildMigrationMatrix(schemas: NonEmptyList[IgluSchema]): Ior[NonEmptyList[IgluSchema], NonEmptyList[OrderedSchemas]] = {
```

Comment:
I seems like a very wise decision to use `Ior` here. If I understand it correct, it can return something in `Left` if some groups in `schemas` contain only one schema and no meaningful migrations can be provided. Which is not the end of the world and application can proceed ignoring those schemas.

Can we just mention this behavior in comments and write couple of tests showing different scenarios.

Reply:
Yep, your understanding is completely correct. I will add more tests and comment as you suggest.